specification/riskiq/Easm/routes.tsp

```diff
@@ -0,0 +1,172 @@
+  @put
+  createOrReplaceDataConnection is Foundations.ResourceOperation<
+    DataConnection,
+    DataConnectionData,
+    DataConnection
+  >;
+  @doc("Delete a data connection with a given dataConnectionName.")
+  deleteDataConnection is Operations.ResourceDelete<DataConnection>;
+}
+
+@tag("Discovery Groups")
+@doc("Provides access to discovery resources.")
+interface DiscoveryGroups {
+  @doc("Retrieve a list of discovery group for the provided search parameters.")
+  listDiscoGroup is Operations.ResourceList<DiscoGroup, DiscoGroupsListTrait>;
+  @doc("Validate a discovery group with a given groupName.")
+  @action("validate")
+  validateDiscoGroup is Operations.ResourceCollectionAction<
+    DiscoGroup,
+    DiscoGroupData,
+    ValidateResult
+  >;
+  @doc("Retrieve a discovery group with a given groupName.")
+  getDiscoGroup is Operations.ResourceRead<DiscoGroup>;
+  #suppress "@azure-tools/typespec-azure-core/use-standard-operations"
+  @doc("Create a discovery group with a given groupName.")
+  @put
+  createOrReplaceDiscoGroup is Foundations.ResourceOperation<
+    DiscoGroup,
+    DiscoGroupData,
+    DiscoGroup
+  >;
+  @doc("Run a discovery group with a given groupName.")
+  @action("run")
+  runDiscoGroup is Operations.ResourceAction<DiscoGroup, {}, NoContentResult>;
+  #suppress "@azure-tools/typespec-azure-core/use-standard-operations"
+  @doc("Retrieve a collection of discovery run results for a discovery group with a given groupName.")
+  @route("/discoGroups/{groupName}/runs")
+  @get
+  listRuns is Foundations.Operation<
+    DiscoGroupsListRunsParameter,
+    DiscoRunPageResult
+  >;
+}
+
+@tag("Discovery Templates")
+@doc("Provides access to disco template resources.")
+interface DiscoveryTemplates {
+  @doc("Retrieve a list of disco templates for the provided search parameters.")
+  listDiscoTemplate is Operations.ResourceList<
+    DiscoTemplate,
+    DiscoTemplatesListTrait
+  >;
+  @doc("Retrieve a disco template with a given templateId.")
+  getDiscoTemplate is Operations.ResourceRead<DiscoTemplate>;
+}
+
+@tag("Reports")
+@doc("Provides access to reporting data.")
+interface Reports {
+  #suppress "@azure-tools/typespec-azure-core/use-standard-operations"
+  @doc("Get billable assets summary for the workspace.")
+  @route("/reports/assets:getBillable")
+  @post
+  getBillable is Foundations.Operation<{}, ReportBillableAssetSummaryResult>;
+  #suppress "@azure-tools/typespec-azure-core/use-standard-operations"
+  @doc("Get the most recent snapshot of asset summary values for the snapshot request.")
+  @route("/reports/assets:getSnapshot")
+  @post
+  getSnapshot is Foundations.Operation<
+    ReportAssetSnapshotRequest,
+    ReportAssetSnapshotResult
+  >;
+  #suppress "@azure-tools/typespec-azure-core/use-standard-operations"
+  @doc("Get asset summary details for the summary request.")
+  @route("/reports/assets:getSummary")
+  @post
+  getSummary is Foundations.Operation<
```

Comment:
Why can't you use the resource action templates here?

Reply:
When I use `Operations.ResourceCollectionAction`, it would result in `/assets:getSummary` instead of `/reports/assets:getSummary`